sources/Outputs/src/Dyd.cpp

```diff
@@ -118,10 +118,18 @@ Dyd::writeHvdcLine(const algo::HvdcLineDefinition& hvdcLine, const std::string&
   auto model = dynamicdata::BlackBoxModelFactory::newModel(hvdcLine.id);
 
   model->setStaticId(hvdcLine.id);
-  if (hvdcLine.converterType == inputs::HvdcLine::ConverterType::LCC) {
-    model->setLib("HvdcPTanPhiDangling");
+  if (hvdcLine.position == algo::HvdcLineDefinition::Position::BOTH_IN_MAIN_COMPONENT) {
+    if (hvdcLine.converterType == inputs::HvdcLine::ConverterType::LCC) {
```

Comment:
If my understandinig is correct we should never be in this case as the hvdcLine is not in the map?

Reply:
Yes you are right. But we want to prepare for the future when we will use a dynamic model for this kind of HVDC lines (with both converters in main cc). The " hvdcLines_.erase(it);" line in the Algo.cpp will be removed so we need to have a prepared structure to handle these HVDC lines.

Reply:
ok!